test/WebJobs.Script.Tests/ProxyMetadataManagerTests.cs

```diff
@@ -49,7 +47,7 @@ public void ProxyMetadata_WhenProxyFileChanges_IsRefreshed()
 
                 Assert.NotSame(proxyMetadata1, proxyMetadata3);
 
-                Assert.Equal(19, proxyMetadata3.Functions.Length);
```

Comment:
@vfedonkin  -- I noticed that this test has been failing since this commit went in: https://github.com/Azure/azure-functions-host/commit/58cf779e3ebef3a31596c35e281492a539050378.

This is expected, right? I think you just missed updating this count here.

Reply:
@brettsam , yes, thank you for fixing that.